From e145459c41b9433dacda655f5b4c5c907a67c45e Mon Sep 17 00:00:00 2001
From: Alan Conway <aconway@apache.org>
Date: Wed, 23 Apr 2014 14:45:13 +0000
Subject: [PATCH 6/7] QPID-5718: Dead code in the HA codebase

Removed: std::ostream& operator<<(std::ostream& o, const QueueSetPrinter& qp)
Already gone: void collectQueueReplicators(const boost::shared_ptr<Exchange>& ex, set<boost::shared_ptr<QueueReplicator> >& collect)

git-svn-id: https://svn.apache.org/repos/asf/qpid/trunk@1589431 13f79535-47bb-0310-9956-ffa450edef68
---
 qpid/cpp/src/qpid/ha/RemoteBackup.cpp | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/qpid/cpp/src/qpid/ha/RemoteBackup.cpp b/qpid/cpp/src/qpid/ha/RemoteBackup.cpp
index 0993c6e..2f4c179 100644
--- a/qpid/cpp/src/qpid/ha/RemoteBackup.cpp
+++ b/qpid/cpp/src/qpid/ha/RemoteBackup.cpp
@@ -84,21 +84,6 @@ RemoteBackup::GuardPtr RemoteBackup::guard(const QueuePtr& q) {
     return guard;
 }
 
-namespace {
-typedef std::set<boost::shared_ptr<broker::Queue> > QS;
-struct QueueSetPrinter {
-    const QS& qs;
-    std::string prefix;
-    QueueSetPrinter(const std::string& p, const QS& q) : qs(q), prefix(p) {}
-};
-std::ostream& operator<<(std::ostream& o, const QueueSetPrinter& qp) {
-    if (!qp.qs.empty()) o << qp.prefix;
-    for (QS::const_iterator i = qp.qs.begin(); i != qp.qs.end(); ++i)
-        o << (*i)->getName() << " ";
-    return o;
-}
-}
-
 void RemoteBackup::ready(const QueuePtr& q) {
     catchupQueues.erase(q);
     if (catchupQueues.size()) {
-- 
1.9.0

